### PR TITLE
LevelDB Config Setup

### PR DESCRIPTION
**Pull request by jackyzha0**
Closes #58

### CODE REVIEW

**Review by RangerMauve — CHANGES_REQUESTED** [review]
Looking good! Just a few comments about structure.

I'm liking how easy the sublevel stuff makes things, and the scehemas are useful to understand the data that's being sent around 💯

File: v1/api/publisher.ts [hunk 1]
```diff
@@ -52,10 +50,8 @@ export async function publisherRoutes (server: FastifyInstance): Promise<void> {
       description: 'Delete a publisher',
       tags: ['publisher']
     }
-  }, async (request, reply) => {
-    // TODO: stub
+  }, async (request, _reply) => {
```
[the remark this answers is not in the document]
Comment (jackyzha0) [reply v1/api/publisher.ts:53]: Yeah mostly just as a reminder that it exists :') I expect we might use it at some point

[the remark this answers is not in the document]
Comment (RangerMauve) [reply v1/api/publisher.ts:53]: Sounds good to me 👍

**Review by RangerMauve — APPROVED** [review]
Looks good to me 👍 Feel free to merge.

Your call on whether you want to merge this into staging first and then staging into v1, or staging into v1 then this into v1.

**Pull request merged**